native/package.json

```diff
@@ -82,7 +82,7 @@
     "react-native-static-server": "^0.5.0",
     "react-native-svg": "^12.1.0",
     "react-native-vector-icons": "^7.1.0",
-    "react-native-webview": "^10.9.1",
+    "react-native-webview": "10.8.2",
```

Comment:
Do we have a bug-task for this? Will hurt us if we try to upgrade native dependencies

Reply:
Will create one :+1: